`npm run jscs` fails on Windows

```
C:\Users\xmr\Desktop\bootstrap>npm run jscs

> bootstrap@4.0.0-alpha.2 jscs C:\Users\xmr\Desktop\bootstrap
> jscs --config=js/.jscsrc js/src js/tests/unit docs/assets/js/src docs/assets/js/ie*.js grunt Gruntfile.js

Error: Path docs/assets/js/ie*.js was not found.
    at .<anonymous> (C:\Users\xmr\Desktop\bootstrap\node_modules\jscs\lib\checker.js:185:19)
    at Array.<anonymous> (C:\Users\xmr\Desktop\bootstrap\node_modules\vow\lib\vow.js:712:39)
    at Immediate.callFns [as _onImmediate] (C:\Users\xmr\Desktop\bootstrap\node_modules\vow\lib\vow.js:23:35)
    at tryOnImmediate (timers.js:543:15)
    at processImmediate [as _immediateCallback] (timers.js:523:5)

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "jscs"
npm ERR! node v6.2.2
npm ERR! npm  v3.9.5
npm ERR! code ELIFECYCLE
npm ERR! bootstrap@4.0.0-alpha.2 jscs: `jscs --config=js/.jscsrc js/src js/tests/unit docs/assets/js/src docs/assets/js/ie*.js grunt Gruntfile.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bootstrap@4.0.0-alpha.2 jscs script 'jscs --config=js/.jscsrc js/src js/tests/unit docs/assets/js/src docs/assets/js/ie*.js grunt Gruntfile.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the bootstrap package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     jscs --config=js/.jscsrc js/src js/tests/unit docs/assets/js/src docs/assets/js/ie*.js grunt Gruntfile.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs bootstrap
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls bootstrap
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\xmr\Desktop\bootstrap\npm-debug.log

```

@cvrebert: why did you switch jscs from grunt-jscs to the standalone package? I mean, the current situation is a mess IMO. We use both standalone packages and grunt wrappers. As you can see, the standalone packages have issues.

--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
   "scripts": {
     "change-version": "node grunt/change-version.js",
     "eslint": "eslint --config js/.eslintrc.json js/src",
-    "jscs": "jscs --config=js/.jscsrc js/src js/tests/unit docs/assets/js/src docs/assets/js/ie*.js grunt Gruntfile.js",
+    "jscs": "jscs --config=js/.jscsrc js/src js/tests/unit docs/assets/js/src grunt Gruntfile.js docs/assets/js/ie-emulation-modes-warning.js docs/assets/js/ie10-viewport-bug-workaround.js",
     "postcss": "postcss --config grunt/postcss.js --replace dist/css/*.css",
     "postcss-docs": "postcss --config grunt/postcss.js --no-map --replace docs/assets/css/docs.min.css && postcss --config grunt/postcss.js --no-map --replace docs/examples/**/*.css",
     "shrinkwrap": "npm shrinkwrap --dev && shx mv ./npm-shrinkwrap.json ./grunt/npm-shrinkwrap.json",

